WordPress/Classes/Services/PostServiceRemote+Concurrency.swift

```diff
@@ -29,6 +30,20 @@ extension PostServiceRemote {
     }
 }
 
+private func mapRemotePostError(error: Error) -> Error {
+    if let error = error as? WordPressAPIError<WordPressComRestApiEndpointError>,
```

Comment:
It's not great, and the naming is not great – I added to the refactoring list.
As long as it's covered with automation – and it is – we should be OK.